src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,18 @@
-export const TodoList = () => {};
+import { Todo } from '../../types/types';
+import { TodoInfo } from '../TodoInfo/TodoInfo';
+
+type TodoListProps = {
+  todos: Todo[];
+};
+
+export const TodoList: React.FC <TodoListProps> = ({ todos }) => (
+  <section className="TodoList">
+    {todos.map(todo => (
+      <TodoInfo
+        todo={todo}
+        key={todo.id}
+        data-id={todo.id}
```

Comment:
You might need to use 'data-testid' attribute instead of 'data-id' for better consistency and testing purposes.